src/main/scala/nl/knaw/dans/easy/fedoratobag/EasyFedoraToBagApp.scala

```diff
@@ -108,20 +108,24 @@ class EasyFedoraToBagApp(configuration: Configuration) extends DebugEnhancedLogg
   }
 
   protected[EasyFedoraToBagApp] def createSecondBag(datasetInfo: DatasetInfo, bagDir1: File, isVersionOf: UUID)(bagDir2: File): Try[Unit] = {
+
     def copy(fileName: String, bag2: DansV0Bag) = {
       (bagDir1 / "metadata" / fileName)
         .inputStream
         .map(addMetadataStreamTo(bag2, fileName))
         .get
     }
 
+    def bagInfoTxt(bag: DansV0Bag) = bag
+      .withEasyUserAccount(datasetInfo.depositor)
+      .withCreated(DateTime.now())
+      .withIsVersionOf(isVersionOf)
+      // the following keys should match easy-fedora-to-bag
+      .addBagInfo("Base-DOI", datasetInfo.doi)
```

Comment:
is this the doi of the current dataset? In the case of `original-versioned` this is correct, but in the case of `fedora-versioned` this should be the Doi of the dataset referenced by the above `isVersionOf`.

idem for the `Base-URN`

Reply:
This method is called `createSecondBag` and only executed in case of `original-versioned`.